ui/components/app/approve-content-card/approve-content-card.js

```diff
@@ -162,27 +162,28 @@ export default function ApproveContentCard({
                     display={DISPLAY.FLEX}
                     flexDirection={FLEX_DIRECTION.COLUMN}
                     alignItems={AlignItems.flexEnd}
-                    textAlign={TEXT_ALIGN.RIGHT}
+                    textAlign={TextAlign.Right}
                   >
                     {useCurrencyRateCheck && (
                       <Box>
                         <Text
                           variant={TextVariant.headingSm}
-                          fontWeight={FONT_WEIGHT.BOLD}
                           color={TextColor.TEXT_DEFAULT}
                           as="h4"
                         >
-                          {formatCurrency(
-                            fiatTransactionTotal,
-                            currentCurrency,
-                          )}
+                          <strong>
+                            {formatCurrency(
+                              fiatTransactionTotal,
+                              currentCurrency,
+                            )}
+                          </strong>
```

Comment:
This is a case where using the <strong> tag isn't necessary because the entire Text element is bold.

Reply:
So we shall remove the `<strong>` tag and move back to `fontWeight={FontWeight.Bold}` ?

Reply:
Shall I undo the changes in a new commit?